### PR TITLE
Update plugin name and identifier to depend on featureset rather than…

### PR DESCRIPTION
**Pull request by lynnerang**
… env vars.

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@
  * @name PLUGIN_IDENTIFIER
  * @type {string}
  */
-const PLUGIN_IDENTIFIER: string = process.env.PLUGIN_IDENTIFIER || 'com.linkedin.figma.specter-plugin';
+const PLUGIN_IDENTIFIER: string = process.env.featureset === 'beta' ? 'com.linkedin.figma.beta-specter-plugin' : 'com.linkedin.figma.specter-plugin';
```
Comment (gblakeman) [inline src/constants.ts:12]: 🥳

Comment (gblakeman) [inline src/constants.ts:12]: 🥳

File: src/assets/css/shared/_all.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@import "node_modules/bourbon/core/bourbon"; // added via devDependencies
+@import url("https://rsms.me/inter/inter.css"); // loads the Inter typeface for us
```
Comment (gblakeman) [inline src/assets/css/shared/_all.scss:2]: Does it make sense to load the typefaces separately in each component? Or should a plugin still have a main/base stylesheet that handles loading the typefaces and a few color/type defaults that are inherited by the rest of the plugins?

**Review by gblakeman — APPROVED** [review]
🎉👊

A question on loading typefaces, but otherwise looks good to me!

File: src/assets/css/shared/_all.scss (hunk 2 above)
Comment (lynnerang) [reply src/assets/css/shared/_all.scss:2]: oh you know, I might not even need to do this year, might just be a product of me trying things to try and weasel out of them errors.  I'll see if axing it in this file and sticking with the original main.scss import does it

Comment (lynnerang) [reply src/assets/css/shared/_all.scss]: removed, thanks!  :)

**Pull request merged**